MAINTAINERS

```diff
@@ -124,8 +124,93 @@ the relevant operators.
 
 * If the change affects the governance, philosophy, goals or principles of the project,
 it must be approved by BDFL.
+
+* A pull request can be in 1 of 5 distinct states, for each of which there is a corresponding label
+that needs to be applied. `Rules.review.states` contains the list of states with possible targets
+for each.
 """
 
+		# Triage
```

Comment:
Yes, that was initially the case, however we'd like to experiment with waffle.io and they have a small limitation with respect to displaying "non-labeled PRs without non-labeled issues". So it's not that big of a deal, and it will be done automatically. If we change our minds, we can always revert to no label here.